.github/workflows/chromatic.yml

```diff
@@ -1,30 +1,28 @@
-name: 'Chromatic'
-
+name: "Chromatic"
 on:
+  workflow_dispatch:
```

Comment:
Sorry I did not notice this before but you have two action triggers here (workflow_dispatch and push) just thought I should ask if this is intentional. Normally we only have one or the other